server/odc-service/src/main/java/com/oceanbase/odc/service/notification/helper/EventBuilder.java

```diff
@@ -254,85 +255,89 @@ private Event ofSchedule(ScheduleEntity schedule, TaskEvent status) {
 
     private <T> void resolveLabels(EventLabels labels, T task) {
         Verify.notNull(labels, "event.labels");
-
-        if (labels.containsKey(CONNECTION_ID)) {
-            try {
-                ConnectionConfig connectionConfig = connectionService.getForConnectionSkipPermissionCheck(
-                        labels.getLongFromString(CONNECTION_ID));
-                labels.put(CLUSTER_NAME, connectionConfig.getClusterName());
-                labels.put(TENANT_NAME, connectionConfig.getTenantName());
-                Environment environment =
-                        environmentService.detailSkipPermissionCheck(connectionConfig.getEnvironmentId());
-                labels.put(ENVIRONMENT, environment.getName());
-            } catch (Exception e) {
-                log.warn("failed to query connection info.", e);
-            }
-        }
         if (labels.containsKey(CREATOR_ID)) {
```

Comment:
Why don't we move this if-brach to the try... finally... block below?

Reply:
because the context is set in this brach. It is the same whether it is placed in finally or not.